docs/field_types.md

```diff
@@ -232,4 +232,12 @@
     </tr>
   </tbody>
 </table>
+<hr/>
+<h2>All fields</h2>
+In case you need to load all available fields in your Presidio deployment, you need to set the `allFields` flag to `true`.
```

Comment:
```suggestion
In case you need Presidio to return all possible PII entities, in your Presidio deployment, you need to set the `allFields` flag to `true`.
```

Reply:
done